src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTest.java

```diff
@@ -3486,4 +3488,30 @@ void openapiTest(VertxTestContext context) {
                 context.completeNow();
             });
     }
+
+    @Test
+    void createConsumerWithGeneratedName(VertxTestContext context) {
+        String groupId = "my-group";
+
+        JsonObject json = new JsonObject();
+        json.put("auto.offset.reset", "earliest");
+        json.put("enable.auto.commit", "true");
+        json.put("fetch.min.bytes", "100");
+
+        postRequest("/consumers/" + groupId)
+                .putHeader("Content-length", String.valueOf(json.toBuffer().length()))
+                .putHeader("Content-type", BridgeContentType.KAFKA_JSON)
+                .as(BodyCodec.jsonObject())
+                .sendJsonObject(json, ar -> {
+                    context.verify(() -> {
+                        assertTrue(ar.succeeded());
+                        HttpResponse<JsonObject> response = ar.result();
+                        assertEquals(HttpResponseStatus.OK.code(), response.statusCode());
+                        JsonObject bridgeResponse = response.body();
+                        String consumerInstanceId = bridgeResponse.getString("instance_id");
+                        assertTrue(consumerInstanceId.startsWith(bridgeID));
```

Comment:
can we avoid the `bridgeID` variable and just using `config.get(BridgeConfig.BRIDGE_ID` ?